mindjunkies/dashboard/forms.py

```
@@ -0,0 +1,9 @@
+from django import forms
+from .models import TeacherVerification, Certificate
+
+class TeacherVerificationForm(forms.ModelForm):
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)